### ISSUE TITLE
Move Extensions activity bar icon to the bottom with the gear menu

### ISSUE THREAD

**Issue by eamodio** [Microsoft/vscode#56013]
I think it would be great to separate the extensions view icon from the others at the top of the activity bar and instead have it at the bottom above the gear menu. To me the extensions are part of vscode like the gear menu, while the other icons are about what I'm working on (or helping along those lines).

**Comment by ramya-rao-a** [comment]
You are right that the rest of the icons help with the day to day workflow and the extensions are more of a customization of  VS Code. To that end, the gear menu already has an entry for it. Look for `Manage Extensions`.

Having said that, I dont believe we will go down the route of getting rid of the extensions icon link from the activity bar altogether any time soon. It is an integral part of the product design. We will definitely re-considers this if the `Mangae Extensions` gear menu item gains more traction or when we consider a major re-design of the product.

Regardless, as a user you can hide the icon. Right click => Hide

**Comment by eamodio** [comment]
To be clear, I don't want the icon removed -- I just want it moved to the bottom of the side bar rather than the top with the other icons.

**Comment by ramya-rao-a** [comment]
My apologies. You clearly said "and instead have it at the bottom above the gear menu" and I totally missed that!

**Comment by eamodio** [comment]
No worries! Thank you!

**Comment by misolori** [comment]
I don't mind this at all, I've seen apps group similar "system" actions together:

![image](https://user-images.githubusercontent.com/35271042/43858910-1bd73184-9b04-11e8-80d7-cc1112b9fbe6.png) ![image](https://user-images.githubusercontent.com/35271042/43858719-96172a2c-9b03-11e8-94e1-2143bae91a78.png)

I'll bring this up with the team.

**Comment by ramya-rao-a** [comment]
cc @kieferrm @chrisdias

**Comment by bpasero** [comment]
-1 for such a change. the top left icons in the activity bar are the toggles for the sidebar. taking some of the icons to a different place while still opening extensions in the side bar seems wrong to me. If we however decided to open extensions in a dedicated view (e.g. modal on top of everything), it would make sense to me.

**Comment by xgdgsc** [comment]
I want the opposite, I feel it should always be at the bottom of the icons. I move it manully every time a new icon is installed.

**Comment by lonix1** [comment]
@eamodio I agree fully. Unsure why turned down (unless it's a technical reason).

- It is related to config
- When lots of icons, it gets very cluttered
- Other apps do this, including MS apps (see image above)

*Did you maybe find a workaround or extension to move those icons around? I looked but found nothing so far.*

**Comment by nms-scribe** [comment]
For others looking for this, there is a simple workaround:

* Right click on the activity bar
* Uncheck the extensions option so that it disappears from the activity bar
* From now on, use the settings gear to open up the extensions interface when needed.

Unfortunately, it reappears when the extensions option is turned on, but at least that's only temporary.

I feel like the arguments against this, and the reason this is closed, are coming from a developer perspective that doesn't match the user perspective. As mentioned by OP, the user POV has the extensions as part of the system, not what we're working on right now. It is only used when you want to change the environment, which means it belongs with the settings. When we are not using it, it gets in the way.

As I understand that not all users are going to agree with me, a better compromise would be to allow users to move the activity buttons on the toolbar where they want them. A user should even be able to remove the "Settings" icon, since the popup menu is also available in the File menu and through Ctrl+Shift+P.
